### PR TITLE
Fix heading offset

### PR DESCRIPTION
**Pull request by ctzsnooze**
The heading value shown in Explorer's IMU calculation did not match that shown in the GPS Heading or IMU Debug values.  

I noted that the blackbox IMU heading calculation is derived from other parameters, and differs from the IMU heading value in debugs.  However it may be useful in some cases, and should at least have the same values and display the same when graphed.  This PR fixes that problem so that all heading values match up numerically and in graphs.

Other minor changes:

- changed 'deg' to the degree symbol, consistently
- put a space before units, consistently, for clarity
- removed unnecessary whitespace
- added Altitude graph configuration
- added a small change to RTH to graph display values, which can be reversed once debugging of maxAlt is complete.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: js/flightlog_fields_presenter.js [hunk 1]
```diff
@@ -833,36 +833,36 @@ function FlightLogFieldPresenter() {
...
             case 'time':
                 return formatTime(value / 1000, true);
-            
+
             case 'gyroADC[0]':
             case 'gyroADC[1]':
             case 'gyroADC[2]':
-                return flightLog.gyroRawToDegreesPerSecond(value / highResolutionScale).toFixed(highResolutionAddPrecision) + " deg/s";
-                
+                return flightLog.gyroRawToDegreesPerSecond(value / highResolutionScale).toFixed(highResolutionAddPrecision) + " °" + '/s';
```
[the remark this answers is not in the document]
Comment (haslinghuis) [reply js/flightlog_fields_presenter.js:855]: Same for line 860, 865, 897, 1011, 1023, 1025

[the remark this answers is not in the document]
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js]: thanks, fixed

File: js/imu.js [hunk 2]
```diff
@@ -104,61 +104,62 @@ function IMU(copyFrom) {
...
-            heading = Math.atan2(headingY, headingX) + magneticDeclination / 10.0 * RAD;
-    
-        if (heading < 0)
-            heading += 2 * Math.PI;
-    
+            heading = Math.atan2(headingY, headingX) + magneticDeclination / 10.0 * RAD; // RAD = pi/180
+
+            heading += 2 * Math.PI; // positive all the time, we want zero to return pi
+            if (heading > 2* Math.PI)
+               heading -= 2* Math.PI;
```
Comment (haslinghuis) [inline js/imu.js:129]:
```suggestion
            if (heading > 2 * Math.PI)
               heading -= 2 * Math.PI;
```

File: js/flightlog_fields_presenter.js [hunk 3]
```diff
@@ -1020,30 +1020,41 @@ function FlightLogFieldPresenter() {
...
                 case 'ESC_SENSOR_RPM':
-                    return value.toFixed(0) + "rpm";
+                    return value.toFixed(0) + " rpm";
                 case 'ESC_SENSOR_TMP':
-                    return value.toFixed(0) + "°C";
+                    return value.toFixed(0) + " °C";
+                case 'ALTITUDE':
+                    switch (fieldName) {
+                        case 'debug[0]': // GPS Trust * 100
+                            value.toFixed(0);
```
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:1044]:
```suggestion
                            return value.toFixed(0);
```

File: js/graph_config.js [hunk 4]
```diff
@@ -910,19 +907,25 @@ GraphConfig.load = function(config) {
...
     GraphConfig.getExampleGraphConfigs = function(flightLog, graphNames) {
+
+        console.log("Running getExampleGraphConfigs");
+
         var
             result = [],
             i, j;
 
+        var sysConfig = flightLog.getSysConfig();
+        var debugModeName = DEBUG_MODE[sysConfig.debug_mode];
```
Comment (haslinghuis) [inline js/graph_config.js:925]:
```suggestion
        const sysConfig = flightLog.getSysConfig();
        const debugModeName = DEBUG_MODE[sysConfig.debug_mode];
```

File: js/imu.js [hunk 5]
```diff
@@ -104,61 +104,62 @@ function IMU(copyFrom) {
...
+
```
Comment (haslinghuis) [inline js/imu.js:130]:
```suggestion
            if (heading > 2 * Math.PI) {
               heading -= 2 * Math.PI;
            }

```

File: js/flightlog_fields_presenter.js (hunk 3 above)
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js:1044]: Thanks, that was a silly mistake.

File: js/graph_config.js (hunk 4 above)
Comment (ctzsnooze) [reply js/graph_config.js:925]: This change was from a commit that was accidentally added in this PR by mistake.  

It was intended to add the debug name into the text that appears to the right of the graph, but it didn't work, so I've removed that commit.

That removes the need to make this change, but I'll put it into the code I was testing.

Thank you again.

File: js/imu.js (hunk 5 above)
Comment (ctzsnooze) [reply js/imu.js:130]: thanks, wasn't sure of the need for the braces

**Comment by ctzsnooze** [comment]
Thanks very much @haslinghuis for your advice and comments.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**